### ISSUE TITLE
Transfer ownership?

### ISSUE THREAD

**Issue by larsbrinkhoff** [eak/trantor#1]
@eak This is yours, right?  Do you want me to transfer ownership of this repository to you?

**Comment by eak** [comment]
I suppose so. I'm not able to do anything with it right now. I take it 
CBF isn't signed up for this stuff?

**Comment by larsbrinkhoff** [comment]
No, I haven't seen CBF on GitHub, and it's not for lack of trying.

**Comment by eswenson1** [comment]
If CBF is Charles Frankston, he is engaged with us on the Multics project. I can contact him if we need to.

**Comment by larsbrinkhoff** [comment]
That's the one. You could just let him know what's going on over here, and ask if he's interested.

**Comment by eswenson1** [comment]
Turns out it was his brother, Bob, who's been following the Multics project. I just ping'ed him for contact info for Charles. I also sent a note to Charles via LinkedIn.

**Comment by cbf66** [comment]
I have a GitHub account.

Mostly for FIRST Robotics stuff. (I'm also not that hard to find via Google!).

**Comment by cbf66** [comment]
But not quite sure why anyone is interested in Trantor. Do you have a need to emulate an ancient CDC Remote Job Entry protocol? Otherwise the next generation project that I started at Livermore (which Earl also contributed to) is probably more interesting, because it was a pure terminal concentrator with Chaos support (and through a gateway Arpanet supported, except it was original pre-IP Arpanet :-)).

**Comment by larsbrinkhoff** [comment]
@budd has show the most interest.

**Comment by larsbrinkhoff** [comment]
Do you know if Eugene Ciccarelli or Richard Lawhorn is around?

**Comment by larsbrinkhoff** [comment]
@philbudne has show the most interest.

**Comment by philbudne** [comment]
mostly academic interest in seeing old things preserved, and available...

**Comment by eswenson1** [comment]
I've sent a message to ECC -- let's see if he joins in.

**Comment by larsbrinkhoff** [comment]
@eak @cbf9427 What was the purpose of TRANTOR?  Which applications did it run?

**Comment by eak** [comment]
Trantor was created by Steve Orszag of the MIT Applied Math department 
to access the CDC and Cray computers at NCAR for his fluid dynamics 
research.  NCAR expected people to access the systems with an expensive 
piece of equipment that read punched cards, sent off the data over a 
synchronous modem using a proprietary protocol, and then sent output 
back to the printer. Not only was it expensive, who wanted to use punch 
cards?  So Orszag bought a PDP-11 and hired undergrads to write software 
to use Emacs-like editing to create programs on the local harddrive, 
submit them using the proprietary protocol (which we sort of had to 
reverse engineer), and print and graph the results. ECC, CBF, and I were 
the early developers (there were a host of undergrad and grad student 
users as well, doing fluid dynamics). Somehow this was cheaper than the 
archaic method. Trantor was then a communications OS with built-in 
applications. We did the development on MIT-MC using the PDP-11 
assembler and emulator.

As CBF mentioned, it is unclear what modern use Trantor would have 
today, other than to amuse its authors by making fun of the source code 
we wrote in the 70s.

I think CBF would be more interested in the CRTSTY program he and I 
spent a lot of time on.

-Earl

**Comment by larsbrinkhoff** [comment]
You mean this?  
https://github.com/PDP-10/its/blob/master/src/syseng/crtsty.1377

**Comment by eak** [comment]
Yes, perhaps CBF would consent to own that.

**Comment by cbf66** [comment]
So what would "owning" CRTSTY mean? Would I be expected to continue adding new terminal types for those who request it? :simple_smile:

Actually, what I'd like to recover is the INFO file that I used to maintain that described the characteristics and control sequences for various terminals. As I recall, that file became fairly well known on the ArapaNet and people from other sites would ftp copies of it for reference. If only I'd thought of putting angle bracket markup on it :-). I believe that Mark Horton grabbed a copy and used it to make termcap (and later terminfo), and acknowledged me somewhere in the source of the same.

**Comment by larsbrinkhoff** [comment]
Would that be INFO; TERMS 160?

            This file attempts to describe a few of the numerous display
    terminals on the market.  If a terminal is supported by either ITS
    directly, or by CRTSTY indirectly, then it should be described herein.
    We may choose to describe other terminals as well.

https://github.com/PDP-10/its/blob/master/doc/info/terms.160

**Comment by cbf66** [comment]
Yup, terms would be it. For some reason I recall actually documenting what the actual control codes were for each terminal, but it would appear I merely documented which features each supported --  shows what one can forget after 35 years or so.. At some point I probably just presumed the CRTSTY source was sufficient documentation.

**Comment by cbf66** [comment]
So if you've got access to ITS files, is there another small pdp11 OS by any chance in the pdp11; or cbf; directories? Might be called switch or something? (Maybe nswit?)

**Comment by larsbrinkhoff** [comment]
@cbf9427 Sorry, no files named SWIT anywhere.  I also briefly checked inside some archive files, but nothing obvious.

But there's still hope.  MIT people are working on preserving the ITS backup tapes.

**Comment by larsbrinkhoff** [comment]
Update: I have a file listing, but not the contents:

CBF; NSWIT @  
CBF; NSWIT BIN  
CBF; NSWIT XFILE  
CBF; NSWIT@ XFILE  

CHSNCP; NSWIT 106  
...  
CHSNCP; NSWIT 274

**Comment by cbfc** [comment]
Yes, those are what I was looking for.

**Comment by cbf66** [comment]
Yes, those are what I was looking for.
